js/adapt-contrib-mcq.js

```diff
@@ -1,12 +1,12 @@
 define([
     'core/js/adapt',
     './mcqView',
-    './mcqModel'
-], function(Adapt, McqView, McqModel) {
+    'core/js/models/itemsQuestionModel'
+], function(Adapt, McqView, ItemsQuestionModel) {
 
     return Adapt.register("mcq", {
         view: McqView,
-        model: McqModel
+        model: ItemsQuestionModel.extend({})
```

Comment:
out of curiosity, why `.extend({})`?

Reply:
It creates a new entry in the inheritance chain. 
Even though we're not explicitly defining any new behaviour for this component type, if someone wanted to add behaviour to the mcq model only they can at this point of the chain.
If we didn't extend here, you'd only have the choice to add functions to the ItemsQuestionModel globally or the Mcq model instance.

Reply:
OK makes sense, could you add a brief comment to that effect?

Reply:
Sure